### ISSUE TITLE
Source Document

### ISSUE THREAD

**Issue by codingSoul289** [SamurAIGPT/EmbedAI#84]
Does the bot access only one source document for answering queries at a time?

**Comment by jannikstdl** [comment]
Yes, only the best one. Seems that this repo is no longer maintained.

I recommend [open-webui](https://github.com/open-webui/open-webui)
